modules/flight_interface/flight_interface.py

```diff
@@ -35,6 +38,13 @@ def create(cls, address: str, timeout_home: float) -> "tuple[bool, FlightInterfa
         # Get Pylance to stop complaining
         assert home_location is not None
 
+        result, flight_interface_logger = logger.Logger.create("flight_interface")
+
+        if result:
+            frame = inspect.currentframe()
+            flight_interface_logger.info("flight interface logger initialized", frame)
+            flight_interface_logger.info(home_location, frame)
```

Comment:
we should probably pass the logger and result into the init in case we want to use the logger throughout the class (we need to pass the result as well because we need to check the result before logging every time)

Reply:
Good point. I was reading thru ur documentation and read that we should just create it in `__init__` so we don't block the process. Is this still true. Link to docs: https://uwarg-docs.atlassian.net/wiki/spaces/CV/pages/2515959830/Logging+Module.

Reply:
After thinking about it, I believe its better to create the logger in `create` and exit if it is `None`. If we create if in `__init__` and its None that means anytime we want to log something we have to do null check which can be easily missed.

Also, if we put it in `create` and don't exit if the logger is `None`, then we would pass in a `logger | None` into init which means that we would still have to do the null check again. Let me know if you agree or have other thoughts.